vizro-core/README.md

```diff
@@ -1,17 +1,125 @@
-# Vizro
+![Vizro Logo Banner - Light](https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/Vizro_Github_Banner_Light_Mode.png#gh-light-mode-only)
+![Vizro Logo Banner - Light](https://raw.githubusercontent.com/mckinsey/vizro/main/.github/images/Vizro_Github_Banner_Dark_Mode.png#gh-dark-mode-only)
```

Comment:
Center align?

Reply:
It seems center aligned when actually rendering, which is strange 😅 , but kedro e.g. has it similar